bin/generic/delete/index.js

```diff
@@ -22,7 +22,7 @@ module.exports = resource => {
             require('bin/_plugins/confirmYes'),
         ],
         resource: resource,
-        options: Object.assign({}, resource.options, options),
+        options: Object.assign({}, options, resource.options),
```

Comment:
why was the order changed?

Reply:
Generic delete overwrites resource.name in options - problem occured because I needed description for ’tag’ to include only ’ID’ and exclude ’or name’

Reply:
changed order allows to fix that

Reply:
probably is better to review that logic across all generic actions. going to mark it as resolved